WebP/project1/application.py

```diff
@@ -42,22 +42,28 @@ def index():
 @app.route("/register", methods=["GET","POST"])
 def register():
 	if (request.method == "POST"):
-		name = request.form.get("user").capitalize()
+		fname = request.form.get("user").capitalize()
 		email = request.form.get("email")
 		password = request.form.get("password")
-		# print(name + ", "+ email + ", " + password)
-		if not name:
+		# print(fname + ", "+ email + ", " + password)
+		if not fname:
 			return render_template("error.html",message="Please provide username")
 		elif not email:
 			return render_template("error.html",message="Please provide email")
 		elif not password:
 			return render_template("error.html",message="Please provide password")
 		else:
 			dt = datetime.datetime.now()
-			user = User(fname=name,email=email,password=password,timestamp=dt)
+			user = User(fname=fname,email=email,password=password,timestamp=dt)
 			db.session.add(user)
 			db.session.commit()
```

Comment:
what if the user already exists?

Reply:
Added a check and displayed it in the error message.